packages/mui-base/src/useList/listActions.types.ts

```diff
@@ -55,6 +56,10 @@ interface ResetHighlightAction {
   event: React.SyntheticEvent | null;
 }
 
+interface ClearSelection {
```

Comment:
All the other actions have the "Action" suffix. Please add it for consistency.

Reply:
Done 👍🏼